docs/components/configurationPanel.md

```diff
@@ -12,9 +12,13 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
 
 # `gamepad.configurationPanel`
 
-This is a `viewComponent` and handles the state and functionality of the configuration panel. It also creates the
-configuration options and stores the custom gamepad configuration for use in the [`inputMapper`](inputMapper.md)
-component.
+This is a [`fluid.viewComponent`](https://tinyurl.com/y4u8xwpd) and creates the configuration options (Action Dropdown,
+Speed Factor Input Box, et cetera) for each gamepad input. This component handles which configuration options should be
+displayed and what list of actions should be available for a given gamepad input. At the same time, the component also
+synchronizes the value of these configuration options with the configuration used by the gamepad and lets you change the
+existing gamepad configuration wither by creating a new gamepad configuration or by switching to the default
```

Comment:
Should be "either".

Reply:
Sorry, that was a typing error. Thanks for pointing this out!